base/uk.ac.stfc.isis.ibex.ui.motor/src/uk/ac/stfc/isis/ibex/ui/motor/views/TableOfMotorsSettingsViewModel.java

```diff
@@ -0,0 +1,39 @@
+package uk.ac.stfc.isis.ibex.ui.motor.views;
+
+import org.eclipse.core.databinding.DataBindingContext;
+import org.eclipse.core.databinding.beans.typed.BeanProperties;
+import uk.ac.stfc.isis.ibex.model.ModelObject;
+import uk.ac.stfc.isis.ibex.motor.internal.MotorsTableSettings;
+
+/**
+ * The view model for the table of motor settings.
+ */
+public class TableOfMotorsSettingsViewModel extends ModelObject {
+    private boolean advancedMinimalMotorView = false;  // Set to false by default for standard view
```

Comment:
I think this should be a singleton because if you click reset layout the simple advanced view selector stops working

Reply:
reset layout now fixed (although it needed more than this model being a singleton)